BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-prerelease-01610-02
+2.0.0-prerelease-01611-05
```

Comment:
nit: There seems to be a weird character at the EOL, at least that's how it looks in CodeFlow. Can you double check to see if that is true?